test/integration/scope-hoisting/es6/re-export-default/b.js

```diff
@@ -1,2 +1,2 @@
-export foo from './c';
+export {default as foo} from './c';
```

Comment:
This was to test export extensions: https://github.com/tc39/proposal-export-default-from

Reply:
Oh sorry, didn't know! Reverted.